presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -309,7 +310,7 @@ public void testRoundTrip(Type type, List<?> readValues)
         testRoundTrip(type, readValues, ImmutableList.of());
     }
 
-    public void testRoundTrip(Type type, List<?> readValues, TupleDomainFilter...filters)
+    public void testRoundTrip(Type type, List<?> readValues, TupleDomainFilter... filters)
```

Comment:
unnecessary change

Reply:
that was auto format when I did cmd+alt+L